openstack/maia/templates/maia-podmonitor.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
+metadata:
+  labels:
+    prometheus: openstack
+  name: prometheus-maia-oprom
+  namespace: maia
+spec:
+  podMetricsEndpoints:
+  - bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+    interval: 30s
+    path: federate
+    port: web
+    scheme: http
+    scrapeTimeout: 25s
+    params:
+      match[]:
+        - '{__name__="up"}'
+  jobLabel: prometheus-maia-oprom-2
```

Comment:
Two things: why does it have `-2`?
I can't see this reflected in the actual job name, is it supposed to do something?

Reply:
also thought something was gonna happen with it, but it doesn't seem so. There is already a ServiceMonitor with this name that queries Prometheus metrics. It has the same name, so I gave this one a `-2` identifier. Will remove it then.